src/main/java/seedu/address/logic/commands/SuggestCommand.java

```diff
@@ -14,7 +14,7 @@
 import seedu.address.model.client.SuggestionType;
 
 /**
- * Suggests a list of clients based on the specified suggestion types
+ * Suggests a list of clients based on the specified suggestion types.
```

Comment:
```suggestion
 * Suggests a list of clients based on the specified suggestion type.
```

Reply:
It should be in the plural because we can have multiple suggestion types being passed in in the form of an ordered set

Reply:
👍